src/normalize.js

```diff
@@ -36,44 +35,55 @@ const parseContentItemContents =
   };
 
 /**
- * Create Gatsby Node structure.
- * @param {Number} nodeId Gebnerated Gatsby node ID.
- * @param {Object} kcArtifact Node's Kentico Kontent data.
- * @param {String} artifactKind Type of the artifact ('item/type')
- * @param {String} codeName Item code name
- * @param {Object} additionalNodeData Additional data
- * @param {Boolean} includeRawContent
- *  Include raw content property in artifact node
- * @return {Object} Gatsby node object
- */
-const createKcArtifactNode =
+* Create Gatsby Node structure.
+* @param {String} nodeId Gebnerated Gatsby node ID.
+* @param {Object} kcArtifact Node's Kentico Kontent data.
+* @param {String} artifactKind Type of the artifact ('item/type')
+* @param {String} codeName Item code name
+* @param {Object} additionalNodeData Additional data
+* @param {Boolean} includeRawContent
+*  Include raw content property in artifact node
+*  @param {Object} rawContent raw content object
+*  used for content property and its digest,
+* @return {Object} Gatsby node object
+*/
+const createKontentArtifactNode =
   (nodeId, kcArtifact, artifactKind, codeName = ``,
-    additionalNodeData = null, includeRawContent = false) => {
-    const nodeContent = stringify(kcArtifact);
+    additionalNodeData = null, includeRawContent = false,
+    rawContent) => {
+    const internal = getNodeInternal(
+      artifactKind,
+      rawContent,
+      includeRawContent,
+      codeName
+    );
+
+    return {
+      ...kcArtifact,
+      ...additionalNodeData,
+      id: nodeId,
+      parent: null,
+      children: [],
+      usedByContentItems___NODE: [],
+      internal,
+    };
+  };
 
+const getNodeInternal =
+  (artifactKind, nodeContent, includeRawContent, codeName) => {
+    // TODO create Content + Content digest from raw data
```

Comment:
Use stingify for `nodeContent` as [here](https://github.com/Kentico/gatsby-source-kontent/pull/105/commits/cf29f74ce74830af8be82233f988c023ffa72e61#diff-3e8c5601b052c6b1ba5be937133b7ddfR74).